infra/charts/management-service/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: v0.6.0
+appVersion: v0.7.0
 description: A Helm chart for Kubernetes Deployment of the XP Managment Service
 name: xp-management
-version: 0.1.0
+version: 0.1.1
```

Comment:
Just wondering, are we just bumping up patch version numbers for the chart versions for each new release?

Reply:
The chart itself has no change - it's just updating the tag value in `values.yaml` with the new app version. That's why.